Odata-docs/changelog/odataconnectedservice.md

```diff
@@ -9,7 +9,66 @@ ms.topic: article
+* [ [#209](https://github.com/OData/ODataConnectedService/pull/209) ] Restore namespace in the metadata document as the default namespace applied in the client proxy
+* [ [#203](https://github.com/OData/ODataConnectedService/pull/203) ] Fix non-compilable code emitted when the option to generate multiple files is checked and the model contains 2 types with the same name but in different namespace
+
+## OData Connected Service 0.12.0
+
+### Improvements and Bug Fixes:
+* [ [#178](https://github.com/OData/ODataConnectedService/pull/178) ] Fix regression errors when generating code for V1-V3 services
+* [ [#178](https://github.com/OData/ODataConnectedService/pull/178) ] Fix regression errors caused by UI updates not running in UI thread
+* [ [#178](https://github.com/OData/ODataConnectedService/pull/178) ] Improve error message when CLR type is not found
+* [ [#183](https://github.com/OData/ODataConnectedService/pull/183) ] Add support for reference model with relative URI
+* [ [#187](https://github.com/OData/ODataConnectedService/pull/187) ] Set more logical default settings
+* [ [#197](https://github.com/OData/ODataConnectedService/pull/197) ] Fix warnings displayed due to empty metadata file created when when the user opts to emit the T4 template on the client project
+
+## OData Connected Service 0.11.1
+
+* [ [#173](https://github.com/OData/ODataConnectedService/pull/173) ] Fixes the signing and strong name issue that caused the Connected Service v0.11.0 not to be loaded by Visual Studio
+
+## OData Connected Service 0.11.0
+
+### New Features:
+* [ [#161](https://github.com/OData/ODataConnectedService/pull/161) ] Adds search box for filtering entities to add on the Schema Types selection page
+* [ [#168](https://github.com/OData/ODataConnectedService/pull/168) ] Generate doc comments in generated code from description annotations in the OData schema
+* [ [#145](https://github.com/OData/ODataConnectedService/pull/145) ] Display the number of selected operation imports
+
+### Improvements and Bug Fixes:
+* [ [#148](https://github.com/OData/ODataConnectedService/pull/148) ] Rename generated parameters source and keys to avoid name collisions with similar names in the model
+* [ [#160](https://github.com/OData/ODataConnectedService/pull/160) ] Fix build error caused by incompatible `System.Text.Json` version with new versions of OData client
+* [ [#135](https://github.com/OData/ODataConnectedService/pull/135) ] Include enum types from enum collection properties in generated code to avoid compilation errors
+* [ [#133](https://github.com/OData/ODataConnectedService/pull/133) ] Fix issue where `false` is generated as `False` (uppercase F) in C# code, causing errors
+* [ [#154](https://github.com/OData/ODataConnectedService/pull/154) ] Fix issue causing undefined Edmx reference to appear in generated code
+
+## OData Connected Service 0.3.1 (Legacy)
```

Comment:
Should this be between 0.10 and 0.11?

Reply:
@wandeg @marabooy The thing is, that release was created by @habbes in between 0.10.0 and 0.11.0. It was not targeted at the active version at the time. I could move it so its appear in the right order version-wise but that order would be misleading chronologically. It appears between 0.10.0 and 0.11.0 in GitHub releases so I'd curious to get an opinion from @habbes (and any other person who might have one) before making the change...

Reply:
For the release notes on MS docs, we don't need the `0.3.1 legacy` notes

Reply:
@gathogojr I think the chronological listing makes sense. But if it's too confusing, then I don't think there's an issue placing it at the bottom. I guess someone looking for a specific version would search the changelog in the order of versions, not chronologically (since they would probably not know the chronological order).

I've also noticed that the pull request you have linked for the 0.3.1 entry is for a different feature/bug fix.

Reply:
@habbes That was a typo. It should be 155 not 115

Reply:
But then again, moving it to the bottom is better that removing it altogether. I have moved it to the bottom.
This argument won it for me: _someone looking for a specific version would search the changelog in the order of versions, not chronologically (since they would probably not know the chronological order)_